src/ui/public/loading_results/loading_results.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+import template from './loading_results.html';
+
+const app = uiModules.get('kibana');
+
+app.directive('loadingResults', function () {
+  return {
+    restrict: 'E',
+    replace: true,
+    template: template,
+    scope: {
+      resultsName: '@',
+    },
```

Comment:
To implement my suggestion, we'd have to add `transclude: true` in here somewhere.